css/main.css

```diff
@@ -0,0 +1,656 @@
+}
+
+.footer-social {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 5vh;
+  filter:
+    invert(67%) sepia(46%) saturate(986%) hue-rotate(179deg)
+    brightness(92%) contrast(82%);
+}
+
+@media all and (min-width: 720px) {
+  .top-social {
+    flex-direction: column;
+    position: absolute;
+    left: 5px;
+    bottom: 35%;
+  }
+
+  .mobile-icon {
+    display: none;
+  }
+
+  .desktop-icon {
+    display: block;
+    height: 50%;
+  }
+
+  #intro {
+    height: 95vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .menu-bars {
+    display: none;
+  }
+
+  #desktop-nav {
+    display: block;
+  }
+
+  #desktop-nav ul {
+    display: flex;
+    margin-right: 10vw;
+  }
+
+  .nav-link {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 20px;
+    color: #fff;
+    padding: 15px;
+    border: 1px solid transparent;
+    transition: 350ms;
+  }
+
+  .nav-link:hover {
+    border: 1px solid #d2d5db;
+  }
+
+  #logo {
+    font-size: 24px;
+  }
+
+  .first-card {
+    display: flex;
+  }
+
+  .first-card-img {
+    width: 40vw;
+  }
+
+  .cards-list {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .card {
+    width: auto;
+    height: 40vh;
+    padding: 2%;
+    cursor: pointer;
+  }
+
+  .card,
+  .card * {
+    transition: all 350ms;
+  }
+
+  .card-link,
+  .card:hover .background-div,
+  .card:hover .card-description {
+    opacity: 0;
+  }
+
+  .card:hover .card-link {
+    opacity: 1;
+  }
+
+  .stacks {
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-around;
+  }
+
+  .about-me-section {
+    align-items: flex-start;
+  }
+
+  .about-me-section > * {
+    max-width: 50vw;
+    margin: 15px 0;
+  }
+
+  #contact {
+    display: flex;
+    padding: 10% 5%;
+  }
+
+  #contact h2 {
+    width: 30vw;
+  }
+
+  #fullname {
+    display: none;
+  }
+
+  #email,
+  textarea {
+    min-width: 50vw;
+  }
+
+  .name-form {
+    display: flex;
+    min-width: 100%;
+  }
+
+  #name,
+  #surname {
+    min-width: 49%;
+  }
+
+  .footer-social {
+    justify-content: center;
+  }
+
+  .social-icon {
+    width: 3vw;
+    height: 32px;
+  }
+}
+
+/* mobile menu */
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  position: absolute;
+  cursor: pointer;
+  background-color: transparent;
+  border: none;
+  right: 1.5rem;
+  transform: translateY(-50%);
+  z-index: 1;
+}
+
+@media (max-width: 768px) {
+  .logo.active {
+    visibility: hidden;
+    transition: 0s;
+  }
+
+  .nav-menu.active {
+    left: 0;
+    margin-top: 20px;
+    background: #3c3a39;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  }
+
+  .nav-menu.active .nav-item {
+    font-weight: 700;
+    line-height: 44px;
+    color: #fff5e1;
+    border-bottom: 1px solid #6f6c6b;
+    padding: 1rem;
+    margin: 0 1.5rem;
+  }
```

Comment:
♻️ Awesome! The mobile menu looks almost identical 🎊. There is something you would need to work on though.

- [ ] Currently, the menu items don't follow the template styling (Color, alignment, size, etc). Even though you put some styles that match the requirements, they are not being applied to your links. One option could be splitting the styling of the menu item from the actual `<a>` tag. You could add the border, padding, etc to the item and the font styles to the `a` tag. 

Currently, your design looks like this:
![image](https://user-images.githubusercontent.com/26911718/204091431-206a4571-525c-4d37-b423-c984bcf8a7ae.png)